test/metabase/driver/mysql_test.clj

```diff
@@ -339,13 +337,8 @@
   (mt/test-driver :mysql
     (if (System/getenv "MB_MYSQL_SSL_TEST_SSL_CERT")
       (testing "MySQL with SSL connectivity using PEM certificate"
-        (let [e      env/env
-              key-fn (fn [k]
-                       (keyword (str/replace-first (name k) "mb-mysql-ssl-test" "mb-mysql-test")))
-              new-e  (m/map-keys key-fn e)]
-          (with-redefs [env/env new-e]
-            ;; with the mysql-test vars having been set to the mysql-ssl variants, run some test to verify connectivity
-            (metabase.query-processor-test.string-extracts-test/test-breakout))))
+        (tu/with-env-keys-renamed-by #(str/replace-first % "mb-mysql-ssl-test" "mb-mysql-test")
+          (metabase.query-processor-test.string-extracts-test/test-breakout)))
```

Comment:
Same thing as before -- alias this in your require and then use the alias here so people know the namespace was required without looking at the `ns` form

Reply:
Done.